libs/test/test.ipkg

```diff
@@ -3,4 +3,6 @@ version = 0.5.1
 
 opts = "--ignore-missing-ipkg"
 
+depends = contrib
```

Comment:
Code of a lib does not change, code of lib's dependencies does not change, but dependencies change. How so? Was it intentional?

Reply:
The dependency was missing.